packages/react-components/src/components/VideoGallery/LocalVideoCameraButton.tsx

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { IButtonStyles, IconButton } from '@fluentui/react';
+import React from 'react';
+import { OptionsDevice } from '../DevicesButton';
+
+export interface LocalVideoCameraButtonProps {
+  /** Array of cameras available to the user. */
+  cameras?: OptionsDevice[];
+  /** Currently selected camera in the local video stream. */
+  currentCamera?: OptionsDevice;
+  /** callback function to change video feed. */
+  cycleCamera: (device: OptionsDevice) => Promise<void>;
+}
+/**
+ * local video tile camera cycle button - for use on mobile screens only.
+ * @param props
+ * @returns
```

Comment:
Remove unused tags like `@returns`. Also `@param props` is kinda useless, so I'd just remove it.